dcf/management/commands/remove_old_items.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding:utf-8 -*-
+
+from django.core.management.base import BaseCommand
+from django.contrib.sites.models import Site
+from django.core.mail import EmailMessage
+from django.utils import timezone
+from django.conf import settings
+from django.db.models import Max
+from django.template import Context
+from django.template.loader import get_template
+from django.utils.translation import ugettext as _
+
+from dcf.models import Item, CustomUser
+
+class Command(BaseCommand):
+    help = 'Remove old items'
+
+    def emailsend(self, recipient, items_list):
+        email_sender = settings.DEFAULT_FROM_EMAIL
+        email_recipient = recipient
+        email_subject = _('Your items have been deleted')
+
+        ctx = {'items': items_list}
+
+        message = get_template('email_remove.html').render(Context(ctx))
+
+        EmailMessage(email_subject, message,
+                        to=[email_recipient],
+                        from_email=email_sender).send()
+
+
+    def handle(self, *args, **options):
+
+        remove_before = timezone.now() \
+            - timezone.timedelta(days=settings.DCF_ITEM_MAX_AGE_DAYS)
+
+        old_items = Item.objects.values('id', 'user_id')\
+           .annotate(last_change=Max('updated'))\
+           .filter(last_change__lte=remove_before.date(), is_active=1)\
```

Comment:
Should use compare with full datetime value, not only date()
`is_active=1` - Should compare with boolean value.